package/src/theme/defaultTheme.js

```diff
@@ -256,6 +256,53 @@ export const rawMuiTheme = {
         }
       }
     },
+    MuiChip: {
+      root: {
+        fontSize: defaultFontSize * 0.875,
+        letterSpacing: 0.28
+      },
+      deletable: {
+        "&:hover": {
+          cursor: "pointer"
+        }
+      },
+      deletableColorPrimary: {
+        "backgroundColor": colors.black02,
+        "border": `1px solid ${colors.black30}`,
+        "color": colors.coolGrey500,
+        "&:hover, &:focus, &:active": {
+          backgroundColor: colors.black05
+        }
+      },
+      deleteIconColorPrimary: {
+        "color": colors.coolGrey,
+        "fontSize": 17,
```

Comment:
This is not one of the font sizes specified here: https://catalyst.reactioncommerce.com/#/Designers/Typography

same issue for the other usages below

Reply:
Yeahhhh... @cassytaylor What should we do about this size 17 icon font? Sizes of Icons are not documented in Typography docs yet. Should we allow this one-off, or create a new variable for a "small size icon font"?

Reply:
Ended up adding a variable for letter-spacing. https://catalyst.reactioncommerce.com/#/Designers/Typography will have to be updated separately, b/c these docs have diverged from the actual styles. I'm gonna work on updating all the letterspacing values next week. Slushweek! #53 #50